Automattic/wp-calypso
# WooCommerce: Add action for deleting products.

This PR adds code that makes it possible to delete a product on the remote site. This does not hook into the UI yet.

Since we are only dealing with deleting a single product at a time for v1 (no bulk), I've opted to keep this simple, but I left a TODO on the action regarding the edit state and storing deletes there for the future. 

To test:
* Run `npm run test-client client/extensions/woocommerce/state/` and make sure all tests pass.

Closes #13627.
